src/MLabsPAB/Files.hs

```diff
@@ -95,6 +120,51 @@ writeAll pabConf policyScripts validatorScripts datums redeemers =
       , mapM (writeRedeemerJsonFile pabConf) redeemers
       ]
 
+readPrivateKeys :: PABConfig -> IO (Either Text (Map PubKeyHash PrivateKey))
+readPrivateKeys pabConf = do
+  files <- listDirectory $ Text.unpack pabConf.pcSigningKeyFileDir
+  let sKeyFiles =
+        map (\filename -> Text.unpack pabConf.pcSigningKeyFileDir ++ "/" ++ filename) $
+          filter (`endsWith` ".skey") files
```

Comment:
Could use `isExtensionOf` here instead, saves a custom def

Reply:
Thanks, changed it as suggested.